docs/src/documenter.md

```diff
@@ -57,5 +57,5 @@ if we set `documenter = true`:
   ```
 
 ### [`Literate.script`](@ref):
-- Documenter style `@ref`s and `@id` will be removed. This means that you can use
-  `@ref` and `@id` in the source file without them leaking to the script.
+- Documenter style `@ref`s, `@extref`s and `@id` will be removed. This means that you can use
+  `@ref`, `@extref` and `@id` in the source file without them leaking to the script.
```

Comment:
This is the only place in the docs where I see the existing functionality mentioned.  

Oddly it is in the context of a keyword argument feature that is deprecated in the source, but the deprecation is not mentioned in the docs.

Reply:
It is discussed a bit here: https://fredrikekre.github.io/Literate.jl/v2/documenter/ but perhaps it can be improved.

Reply:
Yes, that's the page I was looking at.  The first paragraph gave me the impression that this is all dependent on `documenter = (true|false)`, which left me confused until I saw the deprecation notice in the changelog.

> The generators ([Literate.markdown](https://fredrikekre.github.io/Literate.jl/v2/outputformats/#Literate.markdown), [Literate.notebook](https://fredrikekre.github.io/Literate.jl/v2/outputformats/#Literate.notebook) and [Literate.script](https://fredrikekre.github.io/Literate.jl/v2/outputformats/#Literate.script)) supports a keyword argument documenter that lets the generator perform some extra things, keeping in mind that the source code have been written with Documenter.jl in mind. So let's take a look at what will happen if we set documenter = true:

Reply:
Yea you are right. It should be updated to use `Literate.markdown(...; flavor = DocumenterFlavor())`.